tap_mailchimp/schemas/campaigns.json

```diff
@@ -453,6 +464,360 @@
             "null",
             "boolean"
           ]
+        },
+        "can_cancel": {
+          "type": [
+            "null",
+            "boolean"
+          ]
+        },
+        "status": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "emails_sent": {
+          "type": [
+            "null",
+            "integer"
+          ]
+        },
+        "emails_canceled": {
+          "type": [
+            "null",
+            "integer"
+          ]
+        }
+      }
+    },
+    "parent_campaign_id": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "variate_settings": {
+      "type": [
+        "null",
+        "object"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "winning_combination_id": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "winning_campaign_id": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "winner_criteria": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "wait_time": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "test_size": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "subject_lines": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "send_times": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "from_names": {
```

Comment:
I see the datatype is list of strings for this field. Can you please verify? Reference: https://mailchimp.com/developer/marketing/api/campaigns/list-campaigns/